auth/test/clear-sessions-by-user-id.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Utils } from './utils';
+import { TestContainer } from './test-container';
+
+let container: TestContainer;
+
+beforeAll(async () => {
+    container = new TestContainer();
+    await container.ready();
+});
+
+afterEach(async () => {
+    container.user.reset();
+    await container.redis.flushdb();
+});
+
+afterAll(async () => {
+    await container.end();
+});
+
+test('POST clear-sessions-by-user-id', async () => {
+    const admin = await container.request.login();
```

Comment:
No need to save the result in a constant, if you aren't going to use it

Reply:
Done